server/options.go

```diff
@@ -163,6 +169,13 @@ func Metadata(md map[string]string) Option {
 	}
 }
 
+// RegisterCheck run func before registry service
```

Comment:
Can you make this comment match the above

Reply:
yes, sorry. fixed